webcam/webui/package.json

```diff
@@ -19,10 +19,10 @@
     "dependencies": {
         "@types/w3c-image-capture": "^1.0.7",
         "react": "^18.2.0",
-        "taipy-gui": "file:../../venv/lib/python3.10/site-packages/taipy/gui/webapp"
+        "taipy-gui": "file:/Users/gmarabout/.local/share/virtualenvs/demo-face-recognition-3GjZDeuO/lib/python3.11/site-packages/taipy/gui/webapp"
```

Comment:
**note to reviewers** We need a better way to update this (env variable? scripting?). This will come in another PR.